www/class/centreon-clapi/centreon.Config.Poller.class.php

```diff
@@ -543,10 +549,69 @@ public function cfgMove($variables)
                 $msg_copy .= _("OK: All configuration files copied with success.");
             }
         } else {
-            exec("echo 'SENDCFGFILE:" . $host['id'] . "' >> " . $this->centcore_pipe, $stdout, $return);
-            if (!isset($msg_copy)) {
-                $msg_copy = "";
+            /**
+             * Get Parent Remote Servers of the Poller
+             */
+            $statementRemotes = $pearDB->prepare(
+                'SELECT ns.id
+                FROM nagios_server AS ns
+                JOIN platform_topology AS pt ON (ns.id = pt.server_id)
```

Comment:
platform_topology don't existe on centreon-web 20.04.xx, add in 20.10.xx https://github.com/centreon/centreon/blob/98185b835f72e0f414999c3f476d8cb8601bd185/www/install/sql/centreon/Update-DB-20.10.0-beta.1.sql#L28

Clapi over http return status code 500 : 
![image](https://user-images.githubusercontent.com/6943266/111755844-d6336b80-8899-11eb-8224-aab365dda710.png)

Reply:
this is fixed here : https://github.com/centreon/centreon/commit/fbe2f11e02c76556e6f2605be61daffc566d5220
And will be release with 20.04.11 :)